README.md

```diff
@@ -1 +1,40 @@
-# Portfolio
\ No newline at end of file
+# Portfolio
+
+> This is a portfolio repository build from a Figma mobile phone design using CSS Flex box layout and HTML. 
+
+
+## Built With
+
+- HTML
+- CSS
+- Figma
+
+
+## Authors
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/tobuya)
+- Twitter: [@twitterhandle](https://twitter.com/MullerTheGreat1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/thomas-obuya-51b49719b/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
```

Comment:
- [ ] Kindly add the `get started` section to your readme explaining the steps it will take to get a local copy of your repo into the local machine and also running it so that it will be easier for beginners or anyone to make use of your project  👍🏽